types/plotly.js/index.d.ts

```diff
@@ -71,6 +71,13 @@ export interface PlotMouseEvent {
     event: MouseEvent;
 }
 
+export interface PlotHoverEvent {
+    points: PlotDatum[];
+    event: MouseEvent;
```

Comment:
```suggestion
export interface PlotHoverEvent extends PlotMouseEvent {
```
is there a sense to do this that way?

Reply:
Yes, it's better that way 🤗.